extras/friend-lite-sdk/friend_lite/transcribe.py

```
@@ -0,0 +1,235 @@
+                    logger.debug("Timeout waiting for audio chunk, window likely finished.")
+                    break # Exit chunk sending loop
+                except asyncio.CancelledError:
+                    logger.info("Chunk sending task cancelled.")
+                    raise # Re-raise cancellation
+                except Exception as e:
+                    logger.error(f"Error getting/sending audio chunk: {e}", exc_info=True)
+                    raise # Re-raise other exceptions to break segment processing
+
+            # 5. Stop Audio Segment (only if audio was sent in this window)
+            if segment_has_audio:
+                logger.debug(f"Wyoming: Sending AudioStop...")
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)